src/ORM/DataQuery.php

```diff
@@ -1142,16 +1203,51 @@ public function addSelectFromTable($table, $fields)
 
     /**
      * Query the given field column from the database and return as an array.
+     * querying DB columns of related tables is supported but you need to make sure that the related table
+     * is already available in join
+     *
+     * @see DataList::applyRelation()
+     *
+     * example use:
+     *
+     * <code>
+     *  column("<table_name>"."<column_name>")
```

Comment:
I think that this code example should prioritise one like `->column('Title')` since it's probably a more common use case